static/js/server_events_dispatch.js

```diff
@@ -5,7 +5,9 @@ var exports = {};
 exports.dispatch_normal_event = function dispatch_normal_event(event) {
     switch (event.type) {
     case 'alert_words':
+        page_params.alert_words = event.alert_words;
         alert_words.words = event.alert_words;
```

Comment:
I think what we should do instead is `delete page_params.alert_words` after initializing the `alert_words` module, and ensuring that isn't accessed in the frontend code after initialization.  `alert_words.words` is clearly intended to be the authoritative source of what the alert words are.